app/controllers/admin/track_types_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Admin
+  class TrackTypesController < Admin::BaseController
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource :program, through: :conference, singleton: true
+    load_and_authorize_resource :track_type, through: :program
+
+    def index; end
+
+    def edit; end
+
+    def new
+      @track_type = @conference.program.track_types.new
+    end
+
+    def create
+      @track_type = @conference.program.track_types.new(track_type_params)
+      if @track_type.save
+        redirect_to admin_conference_program_track_types_path(conference_id: @conference.short_title),
+                    notice: 'Track type successfully created.'
+      else
+        flash.now[:error] = "Creating track type failed: #{@track_type.errors.full_messages.join('. ')}."
+        render :new
+      end
+    end
+
+    def update
+      if @track_type.update_attributes(track_type_params)
+        redirect_to admin_conference_program_track_types_path(conference_id: @conference.short_title),
```

Comment:
[nitpick] As `@conference` has a `to_param` method the `.short_title` shouldn't be needed.